features/admin.users.v1/components/user-groups-list.tsx

```diff
@@ -0,0 +1,493 @@
+                            </div>
+                        </Header>
+                    );
+                },
+                title: (
+                    <div className="pl-3">
+                        { t("user:updateUser.groups.editGroups.groupList.headers.1") }
+                    </div>
+                )
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "domain",
+                id: "domain",
+                key: "domain",
+                render: (group: GroupsMemberInterface): ReactNode => {
+                    const userGroup: string[] = group?.display?.split("/");
+                    const domain: string = userGroup.length === 1
+                        ? userstoresConfig.primaryUserstoreName
+                        : userGroup[0];
+
+                    return (
+                        <Header
+                            image
+                            as="h6"
+                            className="header-with-icon"
+                            data-componentid={ `${ componentId }-item-heading` }
+                        >
+                            <Label size="mini">
+                                { domain }
+                            </Label>
+                        </Header>
+                    );
+                },
+                title: (
+                    <div className="pl-3">
+                        { t("user:updateUser.groups.editGroups.groupList.headers.0") }
+                    </div>
+                )
+            }
+        ];
+
+        defaultColumns.push(
+            {
+                allowToggleVisibility: false,
+                dataIndex: "action",
+                id: "actions",
+                key: "actions",
+                textAlign: "right",
+                title: ""
+            }
+        );
+
+        return defaultColumns;
+    };
+
+    /**
+     * Resolves data table actions.
+     *
+     * @returns Table actions.
+     */
+    const resolveTableActions = (): TableActionsInterface[] => {
+        const actions: TableActionsInterface[] = [
+            {
+                "data-componentid": `${ componentId }-user-list-item-delete-button`,
+                "data-testid": `${ componentId }-user-list-item-delete-button`,
+                hidden: (): boolean => isReadOnly,
+                icon: (): SemanticICONS =>  "trash alternate",
+                onClick: (e: SyntheticEvent, group: GroupsMemberInterface): void => {
+                    setDeletingGroup(group);
+                    setShowDeleteConfirmationModal(true);
+                },
+                popupText: (): string => t("common:delete"),
+                renderer: "semantic-icon"
+            }
+        ];
+
+        return actions;
+    };
+
+    /**
+     * Shows list placeholders.
+     *
+     * @returns Placeholders.
+     */
+    const showPlaceholders = (): ReactElement => {
+        // When the search returns empty.
```

Comment:
The search result empty check is actually enforced at the place where `showPlaceholders` function is called, isn't it? hence this comment may not be placed here.

Reply:
Fixed with cb0ee2cf165319ca096aa44d358799060f08bbf9